java/daikon/dcomp-dummy/DCRuntime.java.dummy

```diff
@@ -0,0 +1,160 @@
+package java.lang;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+/**
+ * Runtime support for DynComp, a comparability front end for Chicory. This class is a collection of
```

Comment:
I think this class needs to contain exactly the same methods and signatures as the real DCRuntime.  Could you document that correspondence both here and in the real DCRuntime?  Is there any way that a Daikon developer can avoid letting discrepancies creep in, such as extra methods here?